src/sfctl/helps/app_type.py

```diff
@@ -12,14 +12,14 @@
 helps['application provision'] = """
     type: command
     short-summary: Provisions or registers a Service Fabric application type with the
-        cluster using the .sfpkg package in the
-        external store or using the application package in the image store.
+        cluster using the '.sfpkg' package in the external store or using the
+        application package in the image store.
     long-summary:
-        Provisions a Service Fabric application type with the cluster.
-        This is required before any new applications can be instantiated.
-        The provision operation can be performed either on the application package specified
-        by the relativePathInImageStore, or by using the URI of the external .sfpkg.
-        Unless --external-provision is set, this command will expect image store provision.
```

Comment:
We should probably keep this line. I think it makes things a bit more explicit

Reply:
I can add that back.